### ISSUE TITLE
fog below tiles

### ISSUE THREAD

**Issue by tirzah2** [CarlosFdez/world-explorer#31]
hi.
![image](https://github.com/CarlosFdez/world-explorer/assets/83312683/74717fa1-776d-4440-841f-68e796dba9c6)
something strange is happening on my map. v11 foundry. its like the explorer fox is below the tiles instead of on top.

**Comment by CarlosFdez** [comment]
If this is an alignment issue, try updating to 0.8.1. There was a regression if any revealed hex bordered the map.

If this is the tiles is on top of the map thing, that's intentional. Tiles can be used to denote things such as resources or map markers. However, it seems that tiles are sometimes used by people to make maps, so I guess there's merit in perhaps keeping it only below overhead tiles. I'd like some info from people who *do* like putting tiles over the map though before I break everything on them.

**Comment by tirzah2** [comment]
Hi.
I can imagine some uses for having some tiles above the fog, but I use tiles as point of interests and to create a random hexmap. Maybe would it be possible to have a config asking if tiles should be above or below?

**Comment by CarlosFdez** [comment]
This was actually surprisingly difficult to figure out a good default for (I personally do tiles above the map), but I think I came up with a solution. A few months late, but better late than never.

Turns out its possible to set a sort order for canvas layers. By default now (in 0.8.2), the Z-Index is 101. Tiles by default when added are 100, and are rendered underneath. Increasing the ZIndex of the tile *or* decreasing the Z-Index of the entire layer in the options will make the tile render over the overlay.

And it continues to render under tokens and the grid.

![image](https://github.com/CarlosFdez/world-explorer/assets/1286721/68eec17a-0da5-40b3-ac95-7c238e44077f)

![image](https://github.com/CarlosFdez/world-explorer/assets/1286721/a68eb593-be50-41f6-bbbc-dfcfd36fd111)
